kore/test/Test/Kore/Comparators.hs

```diff
@@ -990,13 +990,18 @@ instance EqualWithExplanation SymbolOrAlias where
     compareWithExplanation = structCompareWithExplanation
     printWithExplanation = show
 
-instance SumEqualWithExplanation UnificationError where
-    sumConstructorPair (UnsupportedPatterns a1) (UnsupportedPatterns a2) =
-        SumConstructorSameWithArguments
-        $ EqWrap "UnsupportedPatterns" a1 a2
+instance StructEqualWithExplanation UnificationError where
+    structConstructorName _ = "UnsupportedPatterns"
+    structFieldsWithNames
```

Comment:
We usually match the argument with something like `expected@(UnsupportedPatterns _ _ _)` and use getters below, otherwise we seem to forget to also update this file when adding fields to the struct. See the `SymbolOrAlias` instance above.